### PR TITLE
Ci/use pylint scoring model

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR adds a fix for the cluttering of the code with pylint meta tags with a new pylint job that don't rely on that we fix all pylint suggestions to become green

**How to test availability of pylint job**:
- [x] open the PR travis check of this PR

**Expected test outcome**:
- [x] There should be a build job called "pylint new vs. old code"

**How to test the pylint job**:
- [x] open the job "pylint new vs. old code"

**Expected test outcome**:
- [x] There should be a comparison of the pylint score of the new and the old code 
- [x] The job should have failed since this PR removed all existing pylint warning suppressions

**How to test the linting job**:
- [x] open the job "other linting"

**Expected test outcome**:
- [x] There should be no warnings from pylint (errors are still caught)
- [x] Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by @adrosenbaum 
- [x] tests executed by @patrikgrenfeldt 
- [x] "Merge and deploy" approved by @patrikgrenfeldt 
Thanks for filling in who performed the code review and the test!

This is |minor| **version bump** because extend the functionality with a new pylint job

### CODE REVIEW

File: .gitlint.yaml [hunk 1]
```diff
@@ -89,9 +89,12 @@ pylint:
     - .py
   command: pylint
   arguments:
-    - --rcfile=.configs/pylintrc
+    - --rcfile=.configs/pylintrc'
```
Comment (ingkebil) [inline .gitlint.yaml]:
```suggestion
    - --rcfile=.configs/pylintrc
```

**Comment by patrikgrenfeldt** [comment]
Test job availability:
image

**Comment by patrikgrenfeldt** [comment]
image

This PR will trip over itself on the line that checks for pylint-disablings, that is expected and I don't see a good way around it, I suggest we accept it as it is for the .travis.yml file

File: .travis.yml [hunk 2]
```diff
@@ -39,6 +39,17 @@ jobs:
         - pip install --upgrade pip
         - pip install black
         - git --no-pager diff --name-only --diff-filter=AM $TRAVIS_COMMIT_RANGE | grep ".py" | xargs black --check
+    - name: "pylint score"
+      if: type = pull_request
+      script:
+        - pip install --upgrade pip && pip install -U -r requirements-dev.txt .
+        - ! (grep -r "pylint" . | grep -F "disable=") || echo "Can not run pylint scoring with any pylint warnings disabled, please remove them and try again" && false
```
[the remark this answers is not in the document]
Comment (patrikgrenfeldt) [reply .travis.yml]: fixed

**Pull request merged**